src/main/java/edu/cornell/kfs/gl/CuGeneralLedgerConstants.java

```diff
@@ -5,6 +5,7 @@
 public class CuGeneralLedgerConstants extends GeneralLedgerConstants {
 
 	public static final String ANNUAL_CLOSING_CHARTS_PARAM = "ANNUAL_CLOSING_CHARTS";
+	public static final String COLLECTOR_HEADER_RECORD_TYPE = "HD";
```

Comment:
I'm using this constant when initializing the CollectorBatch object in the builder class. I'm also using it for the unit test code that builds a CollectorBatch from a fixture.